Lib/test/test_pathlib.py

```diff
@@ -744,6 +744,9 @@ def test_as_uri_non_ascii(self):
     def test_match(self):
         P = self.cls
         self.assertFalse(P('A.py').match('a.PY'))
+        self.assertTrue(P('A.py').match('a.PY', case_sensitive=False))
+        self.assertTrue(P('/a/b/c.py').match('/A/*/*.Py', case_sensitive=False))
+        self.assertTrue(P('/a/b/c.py').match('**/*.py', case_sensitive=False))
```

Comment:
I think these should be moved to `test_match_common()`, as they apply on Windows too!

Reply:
Do we need to include them in both the `PurePosixPathTest` and `PureWindowsPathTest` test classes? The `PureWindowsPathTest` class has a `test_match_common` method similar to the `test_match` method in the `PurePosixPathTest` class.

Reply:
I'm referring to `_BasePurePathTest.test_match_common()`, which provides test cases for both `PurePosixPathTest` and `PureWindowsPathTest`.

I think `PureWindowsPathTest.test_match_common()` is misnamed and should be renamed to `test_match()`.